src/awkward/_backends/backend.py

```diff
@@ -19,6 +19,11 @@
 KernelType: TypeAlias = "Callable[..., KernelError | None]"
 
 
+class UfuncLike(Protocol):
+    def __call__(self, *args: ArrayLike, **kwargs) -> ArrayLike:
+        ...
```

Comment:
NumPy ufuncs provide a suite of methods / features that we don't currently support across all backends. I think at this stage, we have no internal use for these other ufuncs and we don't do any packing to ensure that ufuncs only touch "real" data. Therefore, we should only support `ufunc.__call__`; most other things should perhaps be dedicated kernels.

Reply:
It's true that we only ever use `ufunc.__call__`. Awkward v0 used `ufunc.reduceat` to implement `axis=-1` reducers, but none of that code survives.

I just grepped the codebase for all the other ufunc methods. We _only_ use `__call__`, even in NEP-13:

https://github.com/scikit-hep/awkward/blob/33750f362b0398b4d30936a8572cf1c77faa1edf/src/awkward/_connect/numpy.py#L213-L215